repository-data/application/src/main/resources/hcm-config/namespaces/hee/report.yaml

```diff
@@ -191,7 +191,7 @@ definitions:
           /contentBlocks:
             jcr:primaryType: frontend:plugin
             caption: Main content objects
-            compoundList: hee:expanderGroupReference,hee:headingsTOC,hippostd:html,hippogallerypicker:imagelink,hee:insetReference,hee:mediaEmbedReference,hee:richTextReference,hee:statementCardReference,hee:tableReference,hee:tabsReference,hee:warningCalloutReference
+            compoundList: hee:expanderGroupReference,hippostd:html,hippogallerypicker:imagelink,hee:insetReference,hee:mediaEmbedReference,hee:statementCardReference,hee:headingsTOC,hee:tableReference,hee:tabsReference,hee:richTextReference,hee:warningCalloutReference
```

Comment:
Could we move `hee:headingsTOC` (`Table of contents heading`) after `hee:tableReference` (`Table`)?